### DISCUSSION TITLE
Torrent available after its data moved out of container data folder.

### DISCUSSION THREAD

**Discussion by pjotruk** [haugene/docker-transmission-openvpn#2685] [Q&A] [answered]
Hello,
Could somebody help me understand how come that file is available, still being uploaded, after I moved it to a different location, out of docker container data/incomplete folder?
When I pause and resume torrent, or restart a container, it gives me an error to 'check a local files', or something similar. And that is understandable. But until a pause/resume/container restart, torrent does still appear to be actively uploading data. I believe this is easily reproducible. 
Thank you.

**Comment by AnastasiyaSoyka** [comment, marked as answer]
Computer systems are not magic; they operate according to well-understood physical laws.

In this case, in order for the program (Transmission) to know whether or not the torrent data is present, it must ask the operating system to access the filesystem and device memory where the torrent data is stored, and perform a check.

Because this process requires energy, and can take anywhere from microseconds to seconds, depending on the underlying hardware, these checks are only performed when necessary, or when the user requests that they be performed. It is possible to ask the operating system to continuously monitor a location in the filesystem and notify an application when changes are detected, but these systems have the same limitations as above and are oftentimes not used.
